src/components/NavBar.js

```diff
@@ -1,18 +1,21 @@
-import React from "react"
+import React from 'react';
 import { NavLink, Outlet } from 'react-router-dom';
+import logo from '../Assets/logo.png';
+import '../styles/navbar.css';
 
 export default function NavBar() {
   return (
     <>
-      <nav>
-        <div>
+      <nav className="navbar">
+        <div className="logo-name">
+          <img src={logo} alt="space logo" />
           <h1>Space Travelers&#39; Hub</h1>
         </div>
         <ul>
-          <NavLink>Rockets</NavLink>
-          <NavLink>Dragons</NavLink>
-          <NavLink>Mission</NavLink>
-          <NavLink to="my_profile" >My Profile</NavLink>
+          <li><NavLink to="/">Rockets</NavLink></li>
+          <li><NavLink>Dragons</NavLink></li>
+          <li><NavLink>Mission</NavLink></li>
+          <li><NavLink to="my_profile">My Profile</NavLink></li>
```

Comment:
Consider adding `activeClassName="active"` to NavLink

Reply:
Thank you for your clarification @shubhambhoyar077, I had an outdated response from the internet... Then please my only the active one underlined 

> Use <NavLink /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).